src/Upgrader/Business/Upgrader/Bridge/ReleaseGroupTransferBridge.php

```diff
@@ -98,9 +98,21 @@ public function require(ReleaseGroupTransfer $releaseGroup): PackageManagerRespo
      */
     protected function requirePackageCollection(PackageTransferCollection $packageCollection): PackageManagerResponse
     {
-        $requireResponse = $this->packageManager->require($packageCollection);
-        if (!$requireResponse->isSuccess()) {
-            return $requireResponse;
+        $packages = $this->packageCollectionBuilder->filterDevPackage($packageCollection);
```

Comment:
Maybe it's better to name them:
getRequiredPackages and getRequiredDevPackages

Reply:
Thanks, renamed